src/App/Fossa/Config/LicenseScan.hs

```diff
@@ -40,6 +54,25 @@ mergeOpts ::
   ) =>
   a ->
   b ->
-  LicenseScanOpts ->
+  LicenseScanCommand ->
   m LicenseScanConfig
-mergeOpts _ _ (LicenseScanOpts path) = LicenseScanConfig <$> validateDir path
+mergeOpts _ _ (DirectScan path) = RawPathScan <$> collectBaseDir path
+mergeOpts _ _ (FossaDeps path) = VendoredDepsOutput <$> collectBaseDir path
+
+cliParser :: Parser LicenseScanCommand
+cliParser = public <|> private
+  where
+    public = hsubparser fossaDepsCommand
+    private = subparser $ internal <> directScanCommand
+    fossaDepsCommand =
+      command
+        "fossa-deps"
+        ( info (FossaDeps <$> baseDirArg) $
+            progDesc "Like `fossa analyze --output`, but only for native scanning of vendored-dependencies."
+        )
+    directScanCommand =
+      command
+        "direct"
+        ( info (DirectScan <$> baseDirArg) $
+            progDesc "Run a license scan directly on the provided path."
+        )
```

Comment:
I think removing this `direct` argument and making direct license scanning the default behavior would be a more straightforward UX.

Reply:
Oh, I don't think I recognized that one of the commands was hidden and that one of them wasn't. Either way thats fine with me.